src/browser.js

```diff
@@ -14,6 +14,7 @@ class StatusError extends Error {
     this.statusCode = res.status
     this.res = res
     this.responseBody = res.arrayBuffer()
+    this.responseHeaders = {}
```

Comment:
We should be able to get these from the fetch response, no reason we can’t support this in the browser as well.